echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -107,54 +109,41 @@ def test_combine_echodata(raw_datasets):
     eds = [echopype.open_raw(file, sonar_model, xml_file) for file in files]
     combined = echopype.combine_echodata(eds, "overwrite_conflicts")  # type: ignore
 
-    for group_name in combined.group_map:
+    for group_name, value in combined.group_map.items():
         if group_name in ("top", "sonar", "provenance"):
             continue
-        combined_group: xr.Dataset = getattr(combined, group_name)
-        eds_groups = [
-            getattr(ed, group_name)
-            for ed in eds
-            if getattr(ed, group_name) is not None
-        ]
-
-        def union_attrs(datasets: List[xr.Dataset]) -> Dict[str, Any]:
-            """
-            Merges attrs from a list of datasets.
-            Prioritizes keys from later datasets.
-            """
-
-            total_attrs = {}
-            for ds in datasets:
-                total_attrs.update(ds.attrs)
-            return total_attrs
-
-        test_ds = xr.combine_nested(
-            eds_groups,
-            [concat_dims.get(group_name, concat_dims["default"])],
-            data_vars=concat_data_vars.get(
-                group_name, concat_data_vars["default"]
-            ),
-            coords="minimal",
-            combine_attrs="drop",
-        )
-        test_ds.attrs.update(union_attrs(eds_groups))
-        test_ds = test_ds.drop_dims(
-            [
-                "concat_dim",
-                "old_ping_time",
-                "ping_time",
-                "old_time1",
-                "time1",
-                "old_time2",
-                "time2",
-            ],
-            errors="ignore",
-        ).drop_dims(
-            [f"{group}_attrs" for group in combined.group_map], errors="ignore"
-        )
-        assert combined_group is None or test_ds.identical(
-            combined_group.drop_dims(
+        try:
+            combined_group: xr.Dataset = combined[value['ep_group']]
+            eds_groups = [
+                ed[value['ep_group']]
+                for ed in eds
+                if ed[value['ep_group']] is not None
+            ]
+
+            def union_attrs(datasets: List[xr.Dataset]) -> Dict[str, Any]:
+                """
+                Merges attrs from a list of datasets.
+                Prioritizes keys from later datasets.
+                """
+
+                total_attrs = {}
+                for ds in datasets:
+                    total_attrs.update(ds.attrs)
+                return total_attrs
+
+            test_ds = xr.combine_nested(
+                eds_groups,
+                [concat_dims.get(group_name, concat_dims["default"])],
+                data_vars=concat_data_vars.get(
+                    group_name, concat_data_vars["default"]
+                ),
+                coords="minimal",
+                combine_attrs="drop",
+            )
+            test_ds.attrs.update(union_attrs(eds_groups))
+            test_ds = test_ds.drop_dims(
                 [
+                    "concat_dim",
```

Comment:
I was a little puzzled by this "concat_dim" and then I found this line `# xarray inserts this dimension when concatenating along multiple dimensions` [here](https://github.com/OSOceanAcoustics/echopype/blob/6ebc9530eb762ad819cbcb338a2313d05b11e0ac/echopype/echodata/combine.py#L253). Would be good to add this line here to clarify that this is added by xarray, because initially I thought this is maybe a typo or something, since we definitely don't have dimensions named "concat_dim" in echopype. :)